infra/db/prisma/schema.prisma

```diff
@@ -11,8 +11,13 @@ datasource db {
 }
 
 model User {
-  id Int @id @default(autoincrement())
-  email String @unique
-  name String?
-  join_date DateTime?
+  id        Int      @id @default(autoincrement())
+  firstName String
+  lastName  String
+  email     String   @unique
+  password  String
+  address   String?
+  phoneNumber String?
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
```

Comment:
[Should]
Maybe it is possible to use prisma/postgres to take care of this `updateAt` value?
It sohuld automaticly update this value on-change.